ldap/schema/60samba3.ldif

```diff
@@ -1,566 +1,190 @@
-# 60samba3.ldif - Samba v3 schemas
-################################################################################
-#
+## schema file for Fedora/RedHat Directory Server
+##
+## NOTE: this file can be copied as 60samba.ldif into your instance schema
+## directory:
+## cp samba-schema-FDS.ldif /etc/dirsrv/slapd-<instance-name>/schema/60schema.ldif
+##
+## Schema for storing Samba user accounts and group maps in LDAP
+## OIDs are owned by the Samba Team
+##
+## Prerequisite schemas - uid         (cosine.schema)
+##                      - displayName (inetorgperson.schema)
+##                      - gidNumber   (nis.schema)
+##
+## 1.3.6.1.4.1.7165.2.1.x - attributeTypess
+## 1.3.6.1.4.1.7165.2.2.x - objectClasseses
+##
+## Printer support
+## 1.3.6.1.4.1.7165.2.3.1.x - attributeTypess
+## 1.3.6.1.4.1.7165.2.3.2.x - objectClasseses
+##
+## Samba4
+## 1.3.6.1.4.1.7165.4.1.x - attributeTypess
```

Comment:
I'm not sure if it's done intentionally or because of some paste error, but I think it's a mistake to add 's' here (and 'es' to objectClasses)
Example from Samba repo - https://git.samba.org/?p=samba.git;a=blob_plain;f=examples/LDAP/samba.schema

Reply:
Lol, I actually copied this direct from the samba repo, but from the 389 compatible schema. I can fix these typos up though.